### ISSUE TITLE
[FEEDBACK] Application size on screen

### ISSUE THREAD

**Issue by szinn** [Lissy93/dashy#704]
I'm looking at using Dashy to replace my Heimdall dashboard and have a few observations:

When there aren't enough applications to fill out the bottom row, the size of the application buttons are expanded to fill the row as in 
<img width="411" alt="Screen Shot 2022-06-06 at 7 59 59 AM" src="https://user-images.githubusercontent.com/449824/172156797-d7c25464-9f72-48b9-b728-f769520761bc.png">

I'm wondering if there's a way to configure Dashy so that all application buttons are the same size, regardless if the last row is filled or not? (Set them to be a fixed size or perhaps add in "empty non-clickable" entries to fill the row?

Is there a setting to have the configuration panel show up collapsed on load by default?

So far it's pretty awesome - I haven't looked at the widgets yet and I'm hoping to see some live data be able to be associated with the widgets (Ala Heimdall and the *arr apps)

Great work Lissy!

**Comment by szinn** [comment]
Starred repo and re-opened

**Comment by Lissy93** [comment]
Yeah, this should be configurable under `section.displayData`, I think you can set `sectionLayout: grid`. There's also `itemSize` (either `small`, `medium` or `large`) which might help. Other than that, it would be a CSS fix.
Hope that helps :)
